src/formatDate.js

```diff
@@ -50,7 +50,54 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromFormatSeparator = fromFormat[fromFormat.length - 1];
+  const toFormatSeparator = toFormat[toFormat.length - 1];
```

Comment:
As I understand, fromFormat and toFormat always have 4 elements, so you can simplify:
```suggestion
  const fromFormatSeparator = fromFormat[3];
  const toFormatSeparator = toFormat[3];
```